android/PartialSync/app/src/main/java/io/realm/todo/ProjectsActivity.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2018 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.todo;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.EditText;
+
+import java.util.Date;
+
+import io.realm.Realm;
+import io.realm.RealmResults;
+import io.realm.Sort;
+import io.realm.SyncUser;
+import io.realm.todo.model.Project;
+import io.realm.todo.ui.ProjectsRecyclerAdapter;
+
+public class ProjectsActivity extends AppCompatActivity {
+    private Realm mRealm;
```

Comment:
`mRealm`, really? 😉 http://jakewharton.com/just-say-no-to-hungarian-notation/

Reply:
I know, I was just following the previous convention ... I'll change in all tutorials 😏